gateway/mw_http_signature_validation_test.go

```diff
@@ -125,10 +125,7 @@ func waitTimeout(wg *sync.WaitGroup, timeout time.Duration) bool {
 	}
 }
 
-func testPrepareHMACAuthSessionPass(tb testing.TB, hashFn func() hash.Hash, eventWG *sync.WaitGroup, withHeader bool, isBench bool) (string, *APISpec, *http.Request, string) {
-	ts := StartTest(nil)
-	defer ts.Close()
-
+func testPrepareHMACAuthSessionPass(tb testing.TB, ts *Test, hashFn func() hash.Hash, eventWG *sync.WaitGroup, withHeader bool, isBench bool) (string, *APISpec, *http.Request, string) {
```

Comment:
It seems like that new `ts` argument never get used?

Reply:
Sure is! First line of function, one down :arrow_down: :)